CMakeLists.txt

```diff
@@ -358,6 +358,16 @@ if(NOT WITH_CODE_COVERAGE AND NOT MSVC AND NOT CMAKE_C_FLAGS MATCHES "([\\/\\-]O
         CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE}")
 endif()
 
+# Force Visual C++ to use UTF-8
+if(MSVC)
+    if (NOT CMAKE_C_FLAGS MATCHES "[\\/\\-]utf-8")
```

Comment:
Does this require a check to see if it compiles with the flag?

Reply:
All supported Visual C++ versions recognize the flag. IF it errors out, the file is corrupted.